### ISSUE TITLE
Cannot open PCB file with latest (dev) version of pcbnew

### ISSUE THREAD

**Issue by timtashpulatov** [Sakura-IT/decelerator4030#1]
There seem to be a problem opening the PCB with recent versions of pcbnew. Would hate using kicad from year 2013, especially on Linux :)

**Comment by rkujawa** [comment]
Yes, there's a problem, which either is a bug in KiCad, or something specific in this project. It's highly annoying but I didn't find any solution yet. 

Personally I have a CentOS 6 VM for this purpose. Appropriate KiCad version from 2013 is available in EPEL for CentOS/RHEL 6. So I just ssh -X into this VM, which gives near-native experience.

I think I should ask on kicad mailing list about this instead...

**Comment by ghost** [comment]
I also experienced this issue. It cannot find any of the layers.

---

## Error

Error loading board.
IO_ERROR: Layer 'Kleju_Dolna' in file 'C:\Users\stephen\Downloads\decelerator4030-master\board\v0.2\decelerator4030.kicad_pcb' at line 19, is not in fixed layer hash

## from C:/Jenkins/workspace/windows-kicad-msys2-stable/src/kicad-4.0.0-rc1/pcbnew/pcb_parser.cpp : parseLayers() : line 829

## OK

**Comment by ghost** [comment]
Perhaps related to https://forum.kicad.info/t/cant-open-the-demo-with-pcbnew/456

**Comment by rkujawa** [comment]
Yes, exactly, as I mentioned above the current workaround is to use the older KiCad version. Personally I am using:
`$ rpm -q kicad`
`kicad-2013.06.11-1.rev4021.el6.x86_64`

I'll attempt to fix this in pcbnew files "by hand", but don't have the time to do this now.
